### PR TITLE
Package unidecode.0.1.0

### PR DESCRIPTION
**Pull request by sagotch**
### `unidecode.0.1.0`
Convert unicode strings into its ASCII representation

---
* Homepage: https://github.com/geneweb/unidecode
* Source repo: git+https://github.com/geneweb/unidecode.git
* Bug tracker: https://github.com/geneweb/unidecode/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/unidecode/unidecode.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+license: "GNU GPL"
+
+homepage: "https://github.com/geneweb/unidecode"
+
+bug-reports: "https://github.com/geneweb/unidecode/issues"
+
+dev-repo: "git+https://github.com/geneweb/unidecode.git"
+
+depends: [
+  "dune" { build >= "1.1.1" }
```
Comment (mseri) [inline packages/unidecode/unidecode.0.1.0/opam]:
```suggestion
  "dune" { >= "1.1.1" }
```

File: packages/unidecode/unidecode.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "ocaml" { >= "4.05" }
+  ]
+
+build: [ [ "dune" "build"] ]
```
Comment (mseri) [inline packages/unidecode/unidecode.0.1.0/opam]:
```suggestion
build: [ [ "dune" "build" "-p" name "-j" jobs] ]
```

File: packages/unidecode/unidecode.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+install: [ "dune" "install" ]
```
Comment (mseri) [inline packages/unidecode/unidecode.0.1.0/opam]:
```suggestion
```
You don't need this

**Comment by kit-ty-kate** [comment]
The `url` section is missing and thus opam doesn't know where to get the archive from. It seems like you forgot to push the git tag before publishing the release. Alternatively I heard good things about `dune-release` since you use dune.

File: packages/unidecode/unidecode.0.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+license: "GNU GPL"
+
+homepage: "https://github.com/geneweb/unidecode"
+
+bug-reports: "https://github.com/geneweb/unidecode/issues"
+
+dev-repo: "git+https://github.com/geneweb/unidecode.git"
+
+depends: [
+  "dune" { build >= "1.10" }
```
Comment (kit-ty-kate) [inline packages/unidecode/unidecode.0.1.0/opam]:
```suggestion
  "dune" { >= "1.10" }
```
Dune shouldn't be tagged as a build dependency (see #14266)

**Comment by kit-ty-kate** [comment]
The submission is not up to date with the tag
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - unidecode.0.1.0:
            https://github.com/geneweb/unidecode/archive/v0.1.0.tar.gz (Bad checksum, expected md5=253d8a014173780dd93cf2a1e032ef3e)
```

**Comment by sagotch** [comment]
> The submission is not up to date with the tag

Sorry about that. I piped curl to `md5sum` and `sha512sum` but I probably missed some options to get the archive properly.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**